spec/models/solr_document_spec.rb

```diff
@@ -212,4 +213,21 @@
       expect(@mods_doc.mods_xml_for_mods_display).not_to match(/This is a Catalog Heading/)
     end
   end
-end
\ No newline at end of file
+
+  describe "get ocr text" do
+    it "should look for multiple OCR file locations" do
+      doc = SolrDocument.new(:id => "12345",:ocr_id_ss=>'somefilename_99_test.txt')
+      doc.possible_ocr_filenames.should == ['somefilename_99_test.txt','somefilename_test.txt','somefilename_00_test.txt']
```

Comment:
Can we use the `expect` syntax for these tests?  (e.g. `expect(doc.possible_ocr_filenames).to eq ...`)